.github/workflows/linting.yml

```diff
@@ -0,0 +1,28 @@
+name: Linting
+
+on:
+  push:
+    branches:
+      - 'main'
+  pull_request:
+    types: [opened, synchronize, reopened, labeled, unlabeled]
+
+jobs:
+  styles:
+    name: Lint CSS
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+      - name: Use Node LTS version
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - name: Install dependencies
+        run: yarn install
+      - name: Compile assets
+        run: yarn build
+      - name: Lint compiled CSS files
```

Comment:
I'd suggest we run the lint command _only_ on the files edited by the PR: https://github.com/marketplace/actions/list-files-in-pull-request. It only works for pull requests so we could move the linting of the main branch over to the pr-build-and-deploy workflow